packages/augur-ui/src/modules/account/actions/get-rep.ts

```diff
@@ -1,13 +1,19 @@
-import { selectCurrentTimestampInSeconds as getTime } from "store/select-state";
 import logError from "utils/log-error";
 import { AppState } from "store";
 import { Action } from "redux";
 import { ThunkDispatch } from "redux-thunk";
 import { NodeStyleCallback } from "modules/types";
 import { getRep } from "modules/contracts/actions/contractCalls";
+import { updateAssets } from "modules/auth/actions/update-assets";
 
 export default function(callback: NodeStyleCallback = logError) {
-  return (dispatch: ThunkDispatch<void, any, Action>, getState: () => AppState) => {
-    const update = (id: string, status: string) => getRep();
+  return async (dispatch: ThunkDispatch<void, any, Action>, getState: () => AppState) => {
+    await getRep().catch((err: Error) => {
+      console.log("error could not get dai", err);
```

Comment:
Is this `console.log()` supposed to be here?

And it says "could not get **dai**" but the method is `getRep()`.

`logError(new Error("get-Dai"));` shouldn't this be `new Error("get-Rep")`?